utils/tracking.js

```diff
@@ -0,0 +1,24 @@
+import ReactGA from "react-ga"
+import { getCookiesFromLocalStorage } from 'utils';
+
+export const trackPageView = (url) => {
+    if(getCookiesFromLocalStorage && process.env.NEXT_PUBLIC_GA_ID) {
```

Comment:
We only want to send info to GA if the user has agreed to cookies and we're in an environment with the `NEXT_PUBLIC_GA_ID` variable.